README.md

```diff
@@ -1,2 +1,36 @@
-# community
-GoReleaser Community Resources
+# GoReleaser
+
+## Mission statement
+
+TODO
+
+## Code of conduct
+
+Please refer to the [GoReleaser Code of Conduct](CODE_OF_CONDUCT.md)
+
+## Participate and contribute
+
+### Monthly Community meeting and release planning
+
+We maintain a XXX with both our community calls and release planning meeting series.
+
+Join us in our monthly community meetings to engage with the GoReleaser community interactively, to learn about the
+project's directions, and to discuss contributions. Meeting agendas are published in advance - watch out for open PRs on
+the community-meetings/ folder. If you have a pressing issue of common interest you'd like discussed, please let us know
+on Matrix, or simply bring it to the next meeting's Q&A.
```

Comment:
maybe change matrix to discord (and link an invite here?)

Reply:
done